docs/docs/en/guide/task/sql.md

```diff
@@ -28,6 +28,7 @@ Refer to [datasource-setting](../howto/datasource-setting.md) `DataSource Center
 | Custom parameters | SQL task type, and stored procedure is a custom parameter order, to set customized parameter type and data type for the method is the same as the stored procedure task type. The difference is that the custom parameter of the SQL task type replaces the `${variable}` in the SQL statement.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| SQL injection     | You can use SQL injection to change SQL, which will replace `!{Variable}` in the SQL statement. For example, `select * from A where a = !{Variable}`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
```

Comment:
`SQL injection` is a tech term for malicious attack, we should not use it here - https://en.wikipedia.org/wiki/SQL_injection

Reply:
> `SQL injection` is a tech term for malicious attack, we should not use it here - https://en.wikipedia.org/wiki/SQL_injection

How do we define this name? How about naming it ```Sql Interpolation``` or ```Sql Concatenation?```

Reply:
I'm not sure. Something like `dynamic sql` maybe. WDYT? @zhongjiajie

Reply:
> I'm not sure. Something like `dynamic sql` maybe. WDYT? @zhongjiajie

If  use ```dynamic sql ``` to name it, is it better to make it clear in the description of ```Custom parameters``` that the parameter replacement method is precompiled?

Reply:
So as I see, when use `${variable}` it will alway add `"` between our parameter, and `!{Variable}` will use the bare type of parameter like int or string without `"`?

Reply:
> So as I see, when use `${variable}` it will alway add `"` between our parameter, and `!{Variable}` will use the bare type of parameter like int or string without `"`?

yes

Reply:
Hi, @fuchanghai are u still working on this?

Reply:
> Hi, @fuchanghai are u still working on this?

yes， But I don’t know what word is more reasonable，it seem like `SQL injection  ` not reasonable